packages/client/src/views/RegisterForm/duplicate/DuplicateForm.tsx

```diff
@@ -28,8 +39,31 @@ const SubPageContent = styled(Content)`
   max-width: 100%;
 `
 
+const StyledSelect = styled(Select)`
+  width: 192px;
+  ${({ theme }) => theme.fonts.reg18};
+  padding-bottom: 20px;
+  .react-select__control {
+    border-radius: 4px;
+  }
+`
+const StyledText = styled(Text)`
+  padding-bottom: 4px;
+`
+
+const StyledTextArea = styled(TextArea)`
+  border-radius: 4px;
+  margin-bottom: 24px;
+`
+
 export const DuplicateForm = (props: IProps) => {
   const intl = useIntl()
+  const [showModal, setShowModal] = React.useState(false)
+  const [selectedTrackingId, setSelectedTrackingId] = React.useState('')
+  const [isSelected, setIsSelected] = React.useState(false)
```

Comment:
This state is basically if the duplicate tracking id is selected or not. I don't think we really need a state for it as this should always be equal to `Boolean(selectedTrackingId)`